tests/js/sentry-test/performance/initializePerformanceData.ts

```diff
@@ -15,7 +15,7 @@ export interface initializeDataSettings {
   project?: any; // TODO(k-fish): Fix this project type.
   projects?: Project[];
   query?: {};
-  selectedProject?: number | string;
```

Comment:
this was the only weird one, i think the type might be a project but i'm not sure. either way its only for testing